addon/templates/components/polaris-labelled.hbs

```diff
@@ -0,0 +1,37 @@
+<div class={{if labelHidden "Polaris-Labelled--hidden"}}>
+  {{#if label}}
+    <div class="Polaris-Labelled__LabelWrapper">
+      {{#polaris-label id=id hidden=false}}
```

Comment:
totally fine, could have just passed `text` here

Reply:
True, was just following the React component...